plugins/azure/containerregistry/acrCMKEncryption.js

```diff
@@ -0,0 +1,50 @@
+const async = require('async');
+const helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'ACR CMK Encryption Enabled',
+    category: 'Container Registry',
+    domain: 'Containers',
+    description: 'Ensure that Microsoft Azure Container registries have CMK Encryption Enabled.',
+    more_info: 'A customer-managed key gives you the ownership to bring your own key in Azure Key Vault. When you enable a customer-managed key, you can manage its rotations, control the access and permissions to use it, and audit its use.',
+    recommended_action: 'Modify Container registries and enable encryption using cmk.',
```

Comment:
Can we modify the encryption of container registry once its created?

Reply:
This one is resolved. No we cannot change encryption settings once it is created